jetson/nav/stateMachine.cpp

```diff
@@ -218,8 +218,10 @@ void StateMachine::run()
             case NavState::GateTurnToCentPoint:
             case NavState::GateDriveToCentPoint:
             case NavState::GateFace:
-            case NavState::GateShimmy:
             case NavState::GateDriveThrough:
+            case NavState::GateTurnToFarPost: //new
+            case NavState::GateDriveToFarPost: //new
+            case NavState::GateTurnToGateCenter: //new
```

Comment:
These comments don't seem to add much value and they are only temporarily accurate.

This goes for the code throughout.